docs/how-to/installation/docker.txt

```diff
@@ -56,29 +96,99 @@ master with the edited configuration:
+       determinedai/determined-master:VERSION
+
+Note that this references the PostgreSQL and Hasura containers running on
+``determined`` Docker network created above. If you are running PostgreSQL
+externally, specify the hostname for the PostgreSQL server in place of
+``determined-db``.
+
+Determined Agents
+~~~~~~~~~~~~~~~~~
+
+As is the case for the master, Determined agent :ref:`configuration values
+<cluster-configuration>` can come from a file, environment variables, or
+command-line arguments. To get a default configuration file for the agent,
+which contains a listing of the available options and descriptions for them,
+run:
+
+.. code::
+
+    id="$(docker create determinedai/determined-agent:VERSION)"
+    docker cp "$id":/etc/determined/agent.yaml .
+    docker rm "$id"
+
+Then edit the ``agent.yaml`` configuration file as appropriate and start the
+agent with the edited configuration:
+
+.. code::
+
+    docker run \
+        -v /var/run/docker.sock:/var/run/docker.sock \
+        -v "$PWD"/agent.yaml:/etc/determined/agent.yaml \
+        determinedai/determined-agent:VERSION
+
+
+Note that the agent container must bind mount the host's Docker daemon socket.
+This allows the ``determinedai/determined-agent`` container to orchestrate the
+containers that execute trials and other tasks.
 
-The process for starting an agent is analogous, with the caveat that the agent
-container must bind mount the host's Docker daemon socket.  This allows the
-``determinedai/determined-agent`` container to orchestrate the containers that
-execute trials and other tasks:
+If you are providing arguments to the container (e.g.,
+``--master-port`` container argument as opposed to the ``DET_MASTER_PORT``
+environment variable), the agent requires ``run`` as the first argument, e.g.:
 
 .. code::
 
-   docker run -v /var/run/docker.sock:/var/run/docker.sock \
-       -v "$PWD"/agent.yaml:/etc/determined/agent.yaml determinedai/determined-agent:VERSION
+    docker run \
+        -v /var/run/docker.sock:/var/run/docker.sock \
+        -v "$PWD"/agent.yaml:/etc/determined/agent.yaml \
+        determinedai/determined-agent:VERSION \
+        run --master-port=8080
 
-Environment variables and command-line arguments can be passed as usual
-for Docker:
+To start an agent with environment variables instead of a configuration file:
 
 .. code::
 
-   docker run -e DET_DB_HOST=the-db-host determinedai/determined-master:VERSION --db-port=5432
-   docker run -v /var/run/docker.sock:/var/run/docker.sock \
-       -e DET_MASTER_HOST=the-master-host determinedai/determined-agent:VERSION run --master-port=8080
+   docker run \
+       --name determined-agent \
+       --network determined \
+       -e DET_MASTER_HOST=determined-master \
+       -e DET_MASTER_PORT=8080 \
+       determinedai/determined-agent:VERSION
+
+Note that if an agent is running on the same machine as the master, you may use
+``determined-master`` as the ``DET_MASTER_HOST``.
```

Comment:
non-blocking: Add something like "(or whatever the name of the master container is)".